components/eamxx/src/physics/nudging/eamxx_nudging_process_interface.hpp

```diff
@@ -13,6 +13,7 @@
 #include "share/grid/point_grid.hpp"
 #include "share/util/scream_vertical_interpolation.hpp"
 #include "share/util/scream_time_stamp.hpp"
+#include "share/util/scream_universal_constants.hpp"
```

Comment:
IIUC, this is needed for the def fill value in the cpp file. If so, the include should be moved to the cpp file.